app/models/concerns/collection_member.rb

```diff
@@ -36,7 +36,7 @@ def online_label
 
   def parent_collection_params
     self["collection"].map do |collection_id|
-      "id:#{collection_id}"
+      "id:#{collection_id.gsub(/a?(\d+)/, '\1')}"
```

Comment:
Should we left & right anchor this regex?
```suggestion
      "id:#{collection_id.sub(/^a(\d+)$/, '\1')}"
```